### ISSUE TITLE
Configure floorplans in Robin Desk Reservation Software

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#5067]
Configure floorplans for 

- [ ] Barton Oaks 3rd floor

Add Desk
Add Spaces
Add Amenities
Add Policies

- [ ] 8700 Cameron Road

Add Desk
Add Spaces
Add Amenities
Add Policies

- [ ] 8900 Cameron Road

Add Desk
Add Spaces
Add Amenities
Add Policies

**Comment by moorerst** [comment]
Training session with Admin team scheduled for 1/27

Robin has uploaded the floorplans for 8700 & 8900 Cameron Road. Our plan is to modify the Barton Oaks map during a training session on Wednesday. Please also begin planning the desk and spaces layout for Cameron Road so we can begin to update them as well. 
We didn’t discuss this before but it’s important to note that each desk that is added to the floorplan counts against our total desk count for the contract. So we only want to add desk that are relevant. We currently have 94 desks available on the map.
Prior to training session please take some time to review the link which gives some instruction on how to Add Desk To Your Map

**Comment by moorerst** [comment]
There really are only 2 areas, at this time where no one is sitting or has a computer they remote into setup, at 8700:

1.	The cubicles across from where Jaime and Alan were sitting, I believe there are 8 or 9 setup with connectivity. 

2.	The desks along the wall across from the offices  where Jen Duthie, Scott Feldman, etc sit. There are 3 available spaces there, two are still being used I believe. The cubicles between these two spaces do not have connectivity so they would not be an option.

**Comment by dianamartin** [comment]
Configured Barton Oaks, Floor 3 - I realized I saw duplicates because Stephen was also editing. I called him and cleared up that I would make the desks and he's add the amenities. 

Figured out how assign desks in the system. Floor plan map looks good, it's complete I think, needs admin QC, but I think it's done. 
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/8d33630f-18a9-4084-af3a-5507036522c2)
